torch/csrc/api/include/torch/nn/modules/activation.h

```diff
@@ -180,6 +180,25 @@ class TORCH_API LogSoftmaxImpl : public torch::nn::Cloneable<LogSoftmaxImpl> {
 
 TORCH_MODULE(LogSoftmax);
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Softmax2d ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Applies the Softmax2d function element-wise.
+/// See https://pytorch.org/docs/master/nn.html#torch.nn.Softmax2d to learn
+/// about the exact behavior of this module.
+class TORCH_API Softmax2dImpl : public torch::nn::Cloneable<Softmax2dImpl> {
+ public:
+  Softmax2dImpl();
```

Comment:
We can remove the default constructor because it will be created by default :D

Reply:
Fixed it